code/controllers/subsystem/job.dm

```diff
@@ -60,6 +60,30 @@ SUBSYSTEM_DEF(job)
 		if(!job.map_check())	//Even though we initialize before mapping, this is fine because the config is loaded at new
 			testing("Removed [job.type] due to map config")
 			continue
+
+
+		//THIS IS FOR MAPTYPES.
+		//Checks if the maptype is the same as the map
+		//Checks mapexclude.
+		if(job.maptype != SSmaptype.maptype)
+			if(job.maptype != "standard")		//Is the job standard on all maps?
+				if(job.loadalways == FALSE)	//We don't really need this, but still important
+					continue
+
+		if(SSmaptype.maptype in job.mapexclude)
+			if(job.loadalways == FALSE)	//Will the game break if we remove it?
+				continue
+
+			if(job.loadalways == TRUE)	//If it does break, set the jobslots to Zero.
+				job.total_positions = 0
+				job.spawn_positions = 0
```

Comment:
```suggestion
			//If it does break, set the jobslots to Zero.
			job.total_positions = 0
			job.spawn_positions = 0
```

Reply:
No. This is for if the game literally cannot run if the job cannot be unloaded, such as clerk.

Reply:
The code above literally calls `continue` if this statement is false. There's no reason to check AGAIN if the code CAN'T REACH HERE otherwise.